packages/itwinui-react/src/core/Table/types/react-table-types.test.tsx

```diff
@@ -0,0 +1,266 @@
+/**
```

Comment:
Missing copyright header.

Reply:
https://github.com/iTwin/iTwinUI/pull/1514/commits/08d5b6f7553a61240191e02b3ff2545ac57ba062: Added. Not sure why `copyrightLinter.js` didn't do this automatically. But that can be investigated in another PR.